src/Lockedfield.php

```diff
@@ -284,6 +284,27 @@ public function showForm($ID, array $options = [])
     }
 
 
+    /**
+     * Clean locks for an asset
+     *
+     * @param string $itemtype asset type
+     * @param int    $items_id asset ID
+     * @return boolean
+     * @since 10.0.4
+     */
+    public static function releaseLockForAsset($itemtype, $items_id)
+    {
+        global $DB;
+        return $DB->delete(
```

Comment:
Using `$DB->delete()` directly will prevent usage of all `CommonDBTM` pre/post purge methods and will also prevent plugins to use pre/post purge hooks.
You should use `Lockedfield::delete()` method instead.

Reply:
fixed